modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/claims-settled/index.js

```diff
@@ -0,0 +1,463 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClaySelect} from '@clayui/form';
+import React, {useEffect, useState} from 'react';
+
+import DonutChart from '../../../common/components/donut-chart';
+import {
+	getClaimsByPeriodSettled,
+	getPolicies,
+	getProducts,
+	getSettledClaims,
+} from '../../../common/services';
+import {
+	currentDateString,
+	january,
+	lastMonthDate,
+	sixMonthsAgoDate,
+	threeMonthsAgoDate,
+} from '../../../common/utils/dateFormatter';
+
+const PERIOD = {
+	SIX_MONTHS: '3',
+	THIS_MONTH: '1',
+	THREE_MONTHS: '2',
+	YTD: '4',
+};
+
+export default function () {
+	const [selectedFilterDate, setSelectedFilterDate] = useState('4');
```

Comment:
There is a magic number here. Could you change it to something with more context?

Reply:
Done!